griptape/labs/tools/aws_kendra/tool.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core import BaseTool
+from griptape.core.decorators import activity
+from schema import Schema, Literal
+from attr import define, field
+import boto3
+
+@define
+class AwsKendraClient(BaseTool):
+    index_id: str = field(default=None, kw_only=True)
+
+    @activity(config={
+        "name": "query",
+        "description": "Can be used to query an AWS Kendra index.",
+        "schema": Schema({
+            Literal(
+                "queryText",
+                description="The query text passed to the AWS Kendra index.",
+            ): str,
+        })
+    })
+    def query(self, params: dict) -> BaseArtifact:
+        try:
+            kendra = boto3.client('kendra')
+            query_text = params['values']["queryText"]
+            response = kendra.query(IndexId=self.index_id, QueryText=query_text)
+            response_text = ''
+            for query_result in response["ResultItems"]:
```

Comment:
Should we split the result into a `ListArtifact` of `TextArtifact`s? If not, then we should probably separate documents with `\n\n`, so it's easier to chunk later.

Reply:
Didn't know we had a ListArtifact, done!